### PR TITLE
The Hotline: Now on pdictng!

### PR DESCRIPTION
**Pull request by itdaniher**
This will be a long-lived feature branch until @iamalwaysuncomfortable 's payment_monitor refactoring lands. Should be able to shave off a few hundred SLOC off this PR when send_payment just works :tm:!

### CODE REVIEW

File: hotline/clangat.py [hunk 1]
```diff
@@ -0,0 +1,877 @@
...
+        # okay, this now maps the tag (restore key) of each of the above to the instance of the PersistDict class
+        self.state = {
+            self.__getattribute__(attr).tag: self.__getattribute__(attr)
+            for attr in dir(self)
+            if isinstance(self.__getattribute__(attr), aPersistDict)
+        }
+        super().__init__()
+
+    async def get_displayname(self, uuid):
+        uuid = uuid.strip("\u2068\u2069")
```
Comment (technillogue) [inline hotline/clangat.py:131]: can you replace these with the unicode names in \N{...} format?

Comment (itdaniher) [reply hotline/clangat.py:141]: (maybe?)

Comment (technillogue) [reply hotline/clangat.py:106]:
```
In [2]: def expand_unicode(u: str) -> str: return '"' + "".join(r"\N{" + name + "}" for name in map(unicodedata.name, u)) + '"'

In [3]: isolates = "\u2068\u2069"

In [4]: print(expand_unicode(isolates))
"\N{FIRST STRONG ISOLATE}\N{POP DIRECTIONAL ISOLATE}"

In [5]: eval(expand_unicode(isolates)) == isolates
Out[5]: True
```

File: hotline/clangat.py [hunk 2]
```diff
@@ -0,0 +1,1006 @@
...
+            if len(failed):
+                await self.send_message(
+                    msg.uuid,
+                    (
+                        f"failed on\n{[await self.get_displayname(uuid) for uuid in failed]}\n"
+                        "Copy and paste your original pay message and resend to retry."
+                    ),
+                )
+            return "completed sends"
+        return "failed"
```
Comment (technillogue) [inline hotline/clangat.py:422]: when would this ever return "failed"? isn't return "completed sends" unconditional?

File: forest/core.py [hunk 3]
```diff
@@ -472,7 +472,7 @@ async def send_reaction(self, target_msg: Message, emoji: str) -> None:
         await self.outbox.put(cmd)
 
     backoff = False
-    messages_until_rate_limit = 50.0
+    messages_until_rate_limit = 1000.0
     last_update = time.time()
```
Comment (technillogue) [inline forest/core.py:476]: maybe this needs to be configurable/might be inappropriate for signal-cli or something?

**Review by technillogue — APPROVED** [review]

**Pull request merged**